lib/etc/pylint_py3_checker.yaml

```diff
@@ -0,0 +1,8 @@
+allow-shorthand: false
```

Comment:
data files in `lib/etc`? is there a way to avoid shipping this as a file? it will probably not work with some hacks in vsc-install

Reply:
it should be pretty easy to avoid this, and configure it directly by generating your own profile. it's basically just a dict passed to https://github.com/PyCQA/prospector/blob/master/prospector/profiles/profile.py#L38
you should be able to override the profile in the config instance https://github.com/hpcugent/vsc-install/blob/master/lib/vsc/install/commontest.py#L218 (config.profile = ...).